client/src/context/cogeneration.tsx

```diff
@@ -0,0 +1,502 @@
+      const newLessonObjective = lessonObjective;
+      const newEssentialQuestion = lessonEssentialQuestion;
+      const newConclusion = lessonConclusion;
+      const newConceptList = lessonConcepts;
+
+      const newLogs = [
+        ...prev.logPairs,
+        {
+          title: `Generate Lesson Call`,
+          type: "lesson",
+          call: "Call Details",
+          response: "Response Details",
+        },
+      ];
+      const newPrompts = [
+        ...prev.lessonPrompts,
+        {
+          title: `Lesson Generation`,
+          type: "lesson",
+          systemPrompt: lessonSystemPrompt,
+          humanPrompt: lessonHumanPrompt,
+          parseSystemPrompt: lessonParseSystemPrompt,
+          parseHumanPrompt: lessonParseHumanPrompt,
+          parseSystem2: lessonParseSystem2Prompt,
+          parseHuman2: lessonParseHuman2Prompt,
+          tabooSystemPrompt: lessonTabooSystemPrompt,
+          tabooHumanPrompt: lessonTabooHumanPrompt,
+          simulationSystemPrompt: lessonSimulationSystemPrompt,
+          simulationHumanPrompt: lessonSimulationHumanPrompt,
+          slotsSystemPrompt: lessonSlotsSystemPrompt,
+          slotsHumanPrompt: lessonSlotsHumanPrompt,
+          genericSystemPrompt: lessonGenericSystemPrompt,
+          genericHumanPrompt: lessonGenericHumanPrompt,
+          hintFilterSystemPrompt: lessonHintFilterSystemPrompt,
+          hintFilterHumanPrompt: lessonHintFilterHumanPrompt,
+          hintMockPrompt: lessonHintMockPrompt,
+          patchSystemPrompt: lessonPatchSystemPrompt,
+          patchHumanPrompt: lessonPatchHumanPrompt,
+        },
+      ];
+
+      const jsonOutput = prev.jsonOutput
+        ? JSON.parse(prev.jsonOutput)
+        : {
+            lesson_id: "",
+            model: "",
+            temp: "",
+            lesson_name: "",
+            learning_objective: "",
+            introduction: "",
+            essential_question: "",
+            key_concepts: {
+              concept_1: {
+                concept: "",
+                keywords: "",
+              },
+              concept_2: {
+                concept: "",
+                keywords: "",
+              },
+              concept_3: {
+                concept: "",
+                keywords: "",
+              },
+            },
+            hints: {
+              concept_1: {
+                concept: "",
+                hint_1: "",
+                hint_2: "",
+                hint_3: "",
+              },
+              concept_2: {
+                concept: "",
+                hint_1: "",
+                hint_2: "",
+                hint_3: "",
+              },
+              concept_3: {
+                concept: "",
+                hint_1: "",
+                hint_2: "",
+                hint_3: "",
+              },
+            },
+            conclusion: "",
+          };
+      const newJsonOutput = {
+        ...jsonOutput,
+        lesson_id: "3d728b3a-d320-41e5-9091-d40d097e997b",
```

Comment:
hard coded lesson id?  is that going to be a problem?

Reply:
hard coded lesson ID is just mock dummy data until we can actually pull real data from somewhere